src/routers/secure/stream.py

```
@@ -58,3 +69,106 @@ async def stream_events(
+                await upstream_response.aclose()
+
+        return StreamingResponse(
+            stream_iterator(),
+            status_code=upstream_response.status_code,
+            headers=response_headers,
+            media_type=response_headers.get("content-type"),
+        )
+        
+    except HTTPException:
+        raise
+    except Exception as e:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)